File: Quiz_2/Agam_590011213/Quiz_2_Debugging_Question_1.py
def is_digits_only(s):
    if len(s) == 0:
        return False
    for char in s:
        if char < '0' or char > '9':
            return False
    return True

# Example usage
print("Test 1:", is_digits_only("123456"))  # True
print("Test 2:", is_digits_only(""))        # False

File: Quiz_2/Agam_590011213/Quiz_2_Debugging_Question_1.c
#include <stdio.h>
#include <stdbool.h>
#include <string.h>
#include <ctype.h>

bool is_digits_only(const char *s) {
    if (strlen(s) == 0) {
        return false;
    }
    for (int i = 0; s[i] != '\0'; i++) {
        if (s[i] < '0' || s[i] > '9') {
            return false;
        }
    }
    return true;
}

int main() {
    const char *test1 = "123456";
    const char *test2 = "";
    printf("Test 1: %s\n", is_digits_only(test1) ? "True" : "False");
    printf("Test 2: %s\n", is_digits_only(test2) ? "True" : "False");
    return 0;
}
